resources/gnomad_liftover.py

```diff
@@ -0,0 +1,160 @@
+import argparse
+from basics import *
+from gnomad_hail import *
+import logging
+
+
+logging.basicConfig(format="%(levelname)s (%(name)s %(lineno)s): %(message)s")
+logger = logging.getLogger("gnomAD_liftover")
+logger.setLevel(logging.INFO)
+
+
+def checkpoint(ht: hl.Table, dtype: str) -> hl.Table:
+    """
+    Creates a "checkpoint" by writing Table out and reading back in
+    :param Table ht: Table to be evaluated and saved
+    :param str dtype: Data type (exomes or genomes)
+    :return: Table after writing out and reading back in
+    :rtype: Table
+    """
+
+    if dtype == 'exome':
+        ht.write('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_exome_sites.ht', overwrite=True)
+        ht = hl.read_table('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_exome_sites.ht')
+
+    else:
+        ht.write('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_genome_sites.ht', overwrite=True)
+        ht = hl.read_table('gs://seqr-datasets/methods_dev/test_data/kc/gnomAD_genome_sites.ht')
+
+    return ht
+
+
+def lift_ht(ht: hl.Table, dtype: str, rg: hl.genetics.ReferenceGenome) -> hl.Table:
+    """
+    Lifts gnomAD release table from b37 to b38
+    :param Table ht: gnomAD release Table
+    :param str dtype: Data type (exome or genome)
+    :param ReferenceGenome rg: Reference genome, b38
+    :return: gnomAD release Table lifted to b38
+    :rtype: Table
+    """
+
+    # get reference genomes and load liftover chain file
+    #rg37 = hl.get_reference('GRCh37')
+    #rg38 = rg38 = hl.get_reference('GRCh38')
+    #rg37.add_liftover('gs://hail-common/references/grch37_to_grch38.over.chain.gz', rg38)
+
+    # annotate release file with liftover coordinates
+    ht = ht.annotate(new_locus = hl.liftover(ht.locus, rg, include_strand=True),
+                     old_locus=ht.locus
+                    )
+    ht = ht.key_by(locus = ht.new_locus.result)
+    ht = ht.annotate(no_target = hl.is_defined(ht.locus))
```

Comment:
This doesn't seem like it's used anywhere down the line?

Reply:
you're right, I think I added that when I was testing to see how many variants on chr21 failed liftover because there was no liftover locus. will remove